### ISSUE TITLE
Support for Tags

### ISSUE THREAD

**Issue by InsouciantQualms** [justbudget/justbudget#41]
Use Case:

I would like to be able to add zero or more tags to a given transaction.  These would then be filterable in the user interface as well as in any reports.  Example tags:  business, vacation, personal, etc.

**Comment by Rick-Thomas** [comment]
The ability to tag transactions would be a very useful addition...   If you have more than 1 car, the group is Automobile, the category is Gas...  The tag permits me to identify which car is causing the expense without having to create two Gas Categories, one for each car...  Similar examples for lots of categories (insurance, transportation, household, etc)...   I second @shamadoo on the tag addition...

**Comment by Kidglove57** [comment]
I totally agree with this. In my current App I already put tags into the Notes column and search by these. It saves adding yet more sub categories! A dedicated tag column would be so much better.
